app.js

```diff
@@ -35,6 +35,7 @@ app.use((err, req, res, next) => {
   if (err.msg && err.status) {
     res.status(err.status).send({ msg: err.msg });
   }
+  console.log(err);
```

Comment:
You probably don't need this. Looks like it's snuck it's way in 😁

Reply:
That console log played a very vital role in this PR being possible :laughing: I will remove it after merging :D